internal/client/unit_test.go

```diff
@@ -37,7 +37,11 @@ func executeTests(m *testing.M) int {
 	}
 
 	// Create a mock API server listening on the temporary socket
-	fixture := rpmmd_mock.BaseFixture()
+	err = os.Mkdir(tmpdir+"/jobs", 0755)
```

Comment:
`path.Join` would be a bit cleaner but that's just the tiniest nitpick ever.

Reply:
Fixed, because I had to make a change anyway.

Reply:
Since we're picking nits, `filepath.Join` would be more appropriate, but I think we're going to be fine with `path.Join` as well :)